src/components/dashboard/FeedModalList.js

```diff
@@ -90,11 +92,21 @@ const FeedModalList = ({
   )
 
   const feeds = data && data instanceof Array && data.length ? data : undefined
+  let contentOffset = 0
   return (
     <Portal>
       <View style={[styles.horizontalContainer, { opacity: loading ? 0 : 1 }]}>
         <FlatList
           onScroll={({ nativeEvent }) => {
+            if (contentOffset === 0) {
```

Comment:
what are all of this offset calculations for?
if you want it to fire only once use lodash _.once

Reply:
done